cmd/fyne/internal/commands/build.go

```diff
@@ -7,16 +7,90 @@ import (
 	"strings"
 
 	version "github.com/mcuadros/go-version"
+	"github.com/urfave/cli/v2"
 )
 
-type builder struct {
+// Builder generate the executables.
+type Builder struct {
 	os, srcdir, target string
+	goPackage          string
 	release            bool
 	tags               []string
+	tagsToParse        string
 
 	runner runner
 }
 
+// Build returns the cli command for building fyne applications
+func Build() *cli.Command {
+	b := &Builder{}
+
+	return &cli.Command{
+		Name:        "build",
+		Usage:       "Build an application.",
+		Description: "You may specify the --executable to build, but --target will define the OS it is build for indiscriminate of the extention of the executable.",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "target",
+				Aliases:     []string{"os"},
+				Usage:       "The mobile platform to target (android, android/arm, android/arm64, android/amd64, android/386, ios, iossimulator).",
+				Destination: &b.os,
+			},
+			&cli.StringFlag{
+				Name:        "sourceDir",
+				Aliases:     []string{"src"},
+				Usage:       "The directory to package, if executable is not set.",
+				Destination: &b.srcdir,
+			},
+			&cli.StringFlag{
+				Name:        "tags",
+				Usage:       "A comma-separated list of build tags.",
+				Destination: &b.tagsToParse,
+			},
+			&cli.BoolFlag{
+				Name:        "release",
+				Usage:       "Enable installation in release mode (disable debug etc).",
+				Destination: &b.release,
+			},
+			&cli.StringFlag{
+				Name:        "executable",
+				Aliases:     []string{"exe", "o"},
```

Comment:
This just looks wrong - `exe` is input and `o` is output - how can they be aliases?

Reply:
I have updated the documentation on the command to be more precise, but I think it depends on context. In this context, why would you expect that the build command take an executable as input when its goal is to build the executable? I don't see it as a conflict with other command, but actually keep consistency as it means you specify the executable in the same way.

I am fine dropping exe and executable and just use o, just found weird to not mirror other command.

Reply:
You’re right it does not make sense. Which is why I suggested removing it in the first place. If we want to add “-o” that would apply to all the commands not just this one. It is more important to be consistent with this tool than with the go build command when adding functionality. If we want to be more like go compiler then we need to apply that to all commands in this tool.